js/text_encoding.d.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+
+// Typescript already includes type definitions in lib.dom.d.ts
```

Comment:
Actually DOM pulls in webworker, it is defined there:

https://github.com/Microsoft/TypeScript/blob/eaf0d59d354df27dd5915ec814859cdc6322e52a/lib/lib.webworker.d.ts#L2182-L2191

That would be better than pulling in the whole of `lib.dom.d.ts`, but still there is an argument that any overfit of the libraries could end up devaluing the benefits of TypeScript.